CalendarApp/BookingRoom/Controllers/BaseEventController.cs

```diff
@@ -1,14 +1,23 @@
 ﻿using System.Web.Http;
+using Google.Apis.Calendar.v3;
 using GoogleCalendarService;
+using GoogleCalendarService.Manager;
+
 namespace BookingRoom.Controllers
 {
     public class BaseEventController : ApiController
     {
         protected BookingService BookingService;
+        protected EventManager EventManager;
+        protected string TimeZone;
 
-        public BaseEventController(BookingService bookingService)
+        public BaseEventController(BookingService bookingService,EventManager eventManager, CalendarService service)
```

Comment:
We should avoid injecting by type. It is better to extract interface and inject by interface